config/config.go

```diff
@@ -34,6 +35,11 @@ type Metrics struct {
 	EthereumMetricsTick int
 }
 
+// Diagnostics stores diagnostics-related configuration.
+type Diagnostics struct {
+	Port int
+}
```

Comment:
Please add those to the config sample: https://github.com/keep-network/keep-core/blob/master/configs/config.toml.SAMPLE